content/2017/06/27/susy3.rst

```diff
@@ -0,0 +1,871 @@
+and only accepts the second half (context) of the shorthand –
+with or without ``of``:
+
+.. code:: scss
+
+  // Common Use…
+  $padding: gutter();
+
+  // With Context…
+  $padding: gutter(of 4);
+  $same-meaning: gutter(4);
+
+
+Installation & Other Options
+----------------------------
+
+There are full installation instructions
+in the `reference docs`_,
+but you should note that we now provide the Susy API
+with or without prefixes:
+
+.. _reference docs: http://susydocs.oddbird.net/
+
+.. code:: scss
+
+  // unprefixed
+  @import '<path-to>/susy/sass/susy';
+
+  // prefixed
+  @import '<path-to>/susy/sass/susy-prefix';
+
+By default we assume you want Susy without any prefix,
+but importing ``susy-prefix`` will include
+``susy-`` before all function names.
+You can use that if you are worried about name collisions
+with other functions in your project.
+
+
+SVG grids for debugging
+~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want help visualizing and debugging your grids,
+import the SVG Grid Plugin:
+
+.. code:: scss
+
+  // unprefixed
+  @import '<path-to>/susy/sass/plugins/svg-grid';
+
+  // prefixed
+  @import '<path-to>/susy/sass/plugins/svg-grid/prefix';
+
+The plugin adds an ``svg-grid-colors`` setting to your global defaults,
+which you can override in the ``$susy`` settings map.
+It also provides you with a new function,
+``susy-svg-grid()``,
+which will return an inline svg image
+for use on the background of an element:
+
+.. code:: scss
+
+  .container {
+    background: susy-svg-grid() no-repeat scroll;
+  }
+
+SVG grids are much more reliable
+than the old background-image gradient,
+because background gradients have subpixel rounding issues.
+
+
+Building your own Susy system
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Once you get the basics,
+Susy3 also provides tools to help you
+build your own mixins
+and define your own system,
+if that's something you need.
+See the `Plugin Utilities`_
+for more detail.
+
+.. _Plugin Utilities: http://susydocs.oddbird.net/plugin-utils.html
+
+
+Feedback is Always Welcome
+--------------------------
+
+We know we're taking some risks with this release,
+not providing what most people expect from a grid system.
+Some may prefer working with Susy2,
+and that's a solid option as well.
+Over time,
+we hope the CSS Grid module
+will replace all third-party systems.
+
+While we're confident that this is a step forward for Susy,
+we never claim to know what's best for you.
+We'd love your feedback,
+and real-world examples of how you
+make Susy work for you.
+
+We'll keep providing our own tutorials and demos,
+based on the questions we hear most,
+but we also love linking to your
+`sites built with Susy`_,
+and any `third-party tutorials`_
+that we hear about.
+`Contact us`_,
+or `submit a pull request`_.
+You can also
+talk to `SassSusy`_ or `OddBird`_ on twitter,
+or join our `public Slack`_ (with a #Susy channel).
+We're excited to hear from you!
+
+.. _sites built with Susy: http://oddbird.net/susy/sites/
```

Comment:
Yeah, this is the bug David and I were tracking down. That will be the proper URL once I get the redirect fixed...